recipe/meta.yaml

```diff
@@ -11,7 +12,7 @@ source:
   sha256: 8d7f82a9563c0ae46c2725faeca43ee3a2db12a9b75c5a819bcb1b5ef107f322
 
 build:
-  number: 0
+  number: {{ build }}
```

Comment:
Please revert this change. It makes the recipe hard to read and this variable is used only once, so no need for it.

Reply:
This is standard practice for conda-forge build recipes.

Reply:
Nope. We abandoned that a long time ago. Can you please revert?

Reply:
Sure thing - I've reverted this change.